### PR TITLE
Fix native message examples with python3

### PR DESCRIPTION
**Pull request by evolighting**
I/O behave differently between 2 and 3

### CODE REVIEW

**Comment by wbamberg** [comment]
(Note, for more context: this is a fix for https://github.com/mdn/webextensions-examples/issues/156).

Thanks! This makes sense, and seems to work great in Python 3. I wondered if it might be possible to get the same code to work in both 2 and 3? I had a quick look and couldn't see anything, but I'm no expert (it seems like it's basically a matter of being able to read/write raw bytes from/to stdin?).

CC @aswan, whose original Python code I stole for this :).

**Comment by andymckay** [comment]
It should be a matter of just detecting if sys.stdout.buffer is available and then doing a couple of `if` statements based on that and one file would work in both python 2 and 3.

I would also be awesome if we pep-8'd this code, I'll do that in a seperate pull request.

**Comment by wbamberg** [comment]
@andymckay , I have had a go at doing this - would you mind taking a look please? It's tested and appears to work with Pythons 2 and 3.

File: native-messaging/app/ping_pong.py [hunk 1]
```diff
@@ -1,34 +1,62 @@
...
+        sys.stdout.write(encodedMessage['content'])
+        sys.stdout.flush()
 
-while True:
-    receivedMessage = getMessage()
-    if receivedMessage == 'ping':
-        sendMessage(encodeMessage('pong'))
+    while True:
+        receivedMessage = getMessage()
+        if (receivedMessage == "ping"):
```
Comment (andymckay) [inline native-messaging/app/ping_pong.py]: nit: i don't think parentheses are needed in python 2 or 3

**Comment by wbamberg** [comment]
Thanks!

**Pull request merged**